opencensus/trace/trace_state.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import OrderedDict
+import re
+
+_KEY_FORMAT = r'[a-z][_0-9a-z\-\*\/]{0,255}'
+_VALUE_FORMAT = r'[\x20-\x2b\x2d-\x3c\x3e-\x7e]{1,256}'
+
+_KEY_VALIDATION_RE = re.compile('^' + _KEY_FORMAT + '$')
+_VALUE_VALIDATION_RE = re.compile('^' + _VALUE_FORMAT + '$')
+
+
+class TraceState(OrderedDict):
```

Comment:
@adriancole suggested to name this Tracestate to be consistent with the w3c standard. I don't have any strong opinion on this but we should be consistent between libraries (so if we use one name in OpenCensus we should be consistent in all languages, I have not submitted my PR so happy to change if this is the decision).

@SergeyKanzhelev any preferences here between Tracestate vs TraceState?

Reply:
Either one works for me, as long as it is **consistent** across specs and different language implementations.

Reply:
`Tracestate` is preferable

Reply:
Changed to Tracestate.
I've also renamed the file (e.g. trace_state.py to tracestate.py) as we're treating tracestate as a single word.